src/MRA.Jobs.Client/Pages/Applications/Applications.cs

```diff
@@ -81,7 +82,7 @@ protected override async Task OnInitializedAsync()
                 return new { Key = parts[0], Value = parts[1] };
             }).ToDictionary(x => x.Key, x => x.Value);
 
-            _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title");
+            _selectedVacancy.Title = filterDictionary.GetValueOrDefault("Vacancy.Title").Replace("=", "");
```

Comment:
?

Reply:
https://www.notion.so/silkroadprofessionals/Filter-in-the-application-list-6e5c44596ccc4ee5bf54bcf289c23848?pvs=4 
![image](https://github.com/srp-internships/MRA.Jobs/assets/108611175/f1bb905c-1596-4f1d-84dc-17b30c494b7a)